sprint/switch_branch.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# This script allows switching drupal branch, for example, from 8.9.x to 9.0.x
+# or back
+
+set -o errexit
+set -o pipefail
+set -o nounset
+
+if [[ $# != 1 ]]; then
+  echo "Please provide a branch to switch to. For example 'switch_branch.sh 9.0.x'"
+  exit 1
+fi
+
+
+target_branch=$1
+
+pushd drupal8
+set -x
+ddev exec "git stash && git reset --hard && git checkout ${target_branch} && git pull && git stash pop"
+ddev composer require drush/drush:^10
+ddev composer install
+ddev exec drush si --yes standard --account-pass=admin --db-url=mysql://db:db@db/db --site-name=\'Drupal Contribution Time\'
```

Comment:
I think this will error out if the site is already installed, which is the default after running start. A couple of ideas:

* Run `sql:drop` first, but only if a site install detected (`core:status --fields= --format=`)?
* Add to the documentation that should run sql-drop first.

Reply:
Try it, I'm pretty sure `drush si` has always done a drop, it warns you before you do it if you don't do -y.

Reply:
```
$ drush si -y

 // You are about to DROP all tables in your 'db' database. Do you want to
 // continue?: yes.

 [notice] Starting Drupal installation. This takes a while.
```